tests/core/pyspec/eth2spec/test/bellatrix/block_processing/test_process_voluntary_exit.py

```diff
@@ -0,0 +1,71 @@
+from eth2spec.test.context import (
+    spec_state_test,
+    always_bls,
+    with_bellatrix_and_later,
+)
+from eth2spec.test.helpers.keys import pubkey_to_privkey
+from eth2spec.test.helpers.state import (
+    next_epoch,
+)
+from eth2spec.test.helpers.voluntary_exits import (
+    run_voluntary_exit_processing,
+    sign_voluntary_exit,
+)
+
+
+def _run_voluntary_exit_processing_with_specific_fork_version(
```

Comment:
The name seems *slightly* off given that this is specifically for an epoch in the previous signature domain (epoch 0, and a fork at epoch 1). that said, it doesn' matter too too much.

Reply:
maybe a bool here -- `is_before_fork_epoch` -- that says if you utilize a voluntary exit with an epoch pre or post the most recent `state.fork.epoch`. Then we could write a test that has Altair fork_Version but for *after* `fork.epoch` which should fail (and a bellatrix fork_version for before the bellatrix fork)

Reply:
good suggestion 👍  added the new bool flag in https://github.com/ethereum/consensus-specs/pull/2976/commits/6c00b48dca454d1e327b1b81ae7c9493cacb8e60